src/Notification/Notification.module.scss

```diff
@@ -0,0 +1,107 @@
+@import '../mixins.scss';
+@import '../variables.scss';
+
+.wrapper {
+    @include text-normal;
+
+    border-radius: $border-radius;
+    padding: 14px 18px 16px 15px;
+    width: 320px;
+    margin-bottom: 15px;
+}
+
+.default {
```

Comment:
What is the reason to have a separate class, when everything that could be considered **default** might be specified in the **default** `.wrapper` class